ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/DumpFileBuilder.java

```diff
@@ -63,15 +65,20 @@
     protected final Lang lang;
     protected final CompressionStreamFactory compression;
     protected File dumpFile;
+    protected final boolean multipleDump;
+    protected Path path; 
 
     public DumpFileBuilder(int domainId, String[] resourceUriTemplates, String[] accessUriTemplates, Graph[] graphs,
-            Lang lang, CompressionStreamFactory compression) {
+            Lang lang, CompressionStreamFactory compression,boolean multipleDump) {
         this.domainId = domainId;
         this.resourceUriTemplates = resourceUriTemplates;
         this.accessUriTemplates = accessUriTemplates;
         this.graphs = graphs;
         this.lang = lang;
         this.compression = compression;
+        this.multipleDump = multipleDump;
+        this.generateFolder();
+        
```

Comment:
This is not how the builder pattern works. The class should collect necessary data and _only_ when the `build` method is called, it should start to create things. Here, it already starts to create something although the builder might not be done.

Apart from that, it does not make sense to me. Each dump file would still be generated within its own temporary directory. So why is this directory needed at all? :confused: